crossing-repair.lic

```diff
@@ -32,11 +32,13 @@ class CrossingRepair
       fput('swap') unless right_hand
       command = "give #{repairer}"
     else
-      return unless @equipment_manager.get_item?(item)
+      unless item.name.split.last == left_hand.split.last || right_hand.split.last
```

Comment:
This won't work like you want, == has a higher operator precedence than ||.  [Precedence table](https://ruby-doc.org/core-2.2.0/doc/syntax/precedence_rdoc.html)